_includes/store/playmoweb-kimtrack.md

```diff
@@ -0,0 +1,57 @@
+KimTrack is your sleek, native desktop companion for Kimai—right in your menu bar.
+
+Enjoy one-click start/stop and instant project switching, backed by intelligent idle detection and fully customizable preferences to make every second count.
+
+
+### Key Features
+
+- **One-Click Setup**  
+  Get started in seconds. Just download, install, and enter your Kimai API URL and Bearer Token.
+
+- **Native Desktop App**  
+  Enjoy a lightweight, responsive interface on **macOS**, **Windows**, or **Linux**—no browser tabs required. Access all your time-tracking features directly from your menu bar for a seamless experience.
+
+- **Seamless Kimai API Integration**  
```

Comment:
Users do not care about API integrations. They want their data to be up-to-date all the time. 
```suggestion
- **Seamless Kimai Sync**  
```